internal/featuresupport/features.go

```diff
@@ -721,3 +721,41 @@ func (feature *MinimalIso) getFeatureActiveLevel(_ *common.Cluster, infraEnv *mo
 	}
 	return activeLevelNotActive
 }
+
+// MceFeature
+type MceFeature struct{}
+
+func (feature *MceFeature) New() SupportLevelFeature {
+	return &MceFeature{}
+}
+
+func (feature *MceFeature) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDMCE
+}
+
+func (feature *MceFeature) getName() string {
+	return "multicluster engine"
+}
+
+func (feature *MceFeature) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	if !isFeatureCompatibleWithArchitecture(feature, filters.OpenshiftVersion, swag.StringValue(filters.CPUArchitecture)) {
+		return models.SupportLevelUnavailable
+	}
+
+	return models.SupportLevelSupported
+}
+
+func (feature *MceFeature) getIncompatibleFeatures() *[]models.FeatureSupportLevelID {
+	return nil
+}
+
+func (feature *MceFeature) getIncompatibleArchitectures(_ *string) *[]models.ArchitectureSupportLevelID {
+	return &[]models.ArchitectureSupportLevelID{}
+}
```

Comment:
Please verify that there is no CPU architecture limitation (x86, arm, P and Z), if there isn't please return nil instead

```suggestion
func (feature *MceFeature) getIncompatibleArchitectures(_ *string) *[]models.ArchitectureSupportLevelID {
	return nil
}
```

Reply:
I will return nil because all listed architectures are supported for MCE